### PR TITLE
[lmi][docs] update lmi documentation for v12 containers

### PR DESCRIPTION
**Pull request by siddvenk**
## Description ##

Updates the lmi docs for v12 - i'm raising this to the master branch so that changes for v13 get all the required changes/updates from v12 as well.

This needs to be raised to the 0.30.0-dlc branch.

I've pulled in the changes for TRTLLM from Roberts PR as well https://github.com/deepjavalibrary/djl-serving/pull/2517/files

### CODE REVIEW

File: serving/docs/lmi/tutorials/trtllm_aot_tutorial.md [hunk 1]
```diff
@@ -42,7 +45,7 @@ Refer [here](https://github.com/aws/deep-learning-containers/blob/master/availab
 For example:
 
 ```
-docker pull 763104351884.dkr.ecr.us-east-1.amazonaws.com/djl-inference:0.29.0-tensorrtllm0.11.0-cu124
+docker pull 763104351884.dkr.ecr.us-east-1.amazonaws.com/djl-inference:0.23.0-tensorrtllm0.12.0-cu125
```
Comment (ethnzhng) [inline serving/docs/lmi/tutorials/trtllm_aot_tutorial.md:48]:
```suggestion
docker pull 763104351884.dkr.ecr.us-east-1.amazonaws.com/djl-inference:0.30.0-tensorrtllm0.12.0-cu125
```

File: serving/docs/lmi/user_guides/trt_llm_user_guide.md [hunk 2]
```diff
@@ -11,24 +11,23 @@ TRT-LLM LMI supports two options for model artifacts
...
+* Mistral
+* Mixtral 
+* Qwen 
+* GPT2/SantaCoder/StarCoder/GPTBigCode
+* Phi2 
+* OPT 
+* Gemma 
+* T5
+
+TRT-LLM LMI v12 0.30.0 containers come with [TRT-LLM 0.11.0](https://github.com/NVIDIA/TensorRT-LLM/releases/tag/v0.12.0).
```
Comment (ethnzhng) [inline serving/docs/lmi/user_guides/trt_llm_user_guide.md:29]:
```suggestion
TRT-LLM LMI v12 0.30.0 containers come with [TRT-LLM 0.12.0](https://github.com/NVIDIA/TensorRT-LLM/releases/tag/v0.12.0). 
```

**Review by sindhuvahinis — APPROVED** [review]

File: serving/docs/lmi/tutorials/trtllm_aot_tutorial.md (hunk 1 above)
Comment (siddvenk) [reply serving/docs/lmi/tutorials/trtllm_aot_tutorial.md:48]: good catch

File: serving/docs/lmi/user_guides/trt_llm_user_guide.md (hunk 2 above)
Comment (siddvenk) [reply serving/docs/lmi/user_guides/trt_llm_user_guide.md:29]: good catch

**Pull request merged**